indico/web/templates/forms/_person_link_widget_base.html

```diff
@@ -68,6 +72,9 @@
                 secondaryAuthor: {{ field.require_secondary_author | default(false) | tojson }},
                 submitter: {{ field.require_submitter_author | default(false) | tojson }},
                 speaker: {{ field.require_speaker_author | default(false) | tojson }}
+            },
+            sessionUser: {
+                email: {{ session.user.email | default(false) | tojson }}
```

Comment:
No need for the default - if you see this field you are always logged in, so the value won't ever be undefined.

Reply:
Great - will do this